sanghyeok/LV_1/문자열_내_p와_y의_개수.js

```diff
@@ -0,0 +1,13 @@
+function solution(s){
+    let answer = true
+    let Pcount= 0;
+    let Ycount= 0;
+    let str = s.toUpperCase().split('') 
+for(let i =0; i < str.length; i++){
+    if(str[i] ==='P') Pcount++;
+
+    else if(str[i] ==='Y') Ycount++
+}
+    if(Pcount != Ycount) answer = false;
+     return answer;
+}
```

Comment:
```suggestion
function solution(s){
    let pCount = 0;
    let yCount = 0;
    let upperString = s.toUpperCase().split('') 
    for(let i = 0; i < upperString.length; i++){
    if(upperString[i] ==='P') pCount++;
    else if(upperString[i] ==='Y') yCount++;
}
    if(pCount != yCount) return false;
     return true;
}
```
- 컨벤션에 맞지않는 부분이 너무 많이 보이는거 같습니다 혹시 프리터를 이용해 정리를 하셨을까요?? 다음엔 꼭 정렬 후 올려 주시기 바랍니다~!
- 공백이 for문만 뒤로 들어가 있습니다. 변수한테 = 가 붙어있고 for문의 i는 또 떨어져서 값에 붙어있습니다. else if 문 세미클론 빠져있습니다.
- 변수명은 대문자 문자열이므로 좀 더 명확하게 짓는게 좋을것 같습니다!
- 변수명 첫글자는 대문자 사용하지 않습니다!!
- 굳이 반환값인 불리언을 변수로 넣을 필요는 없을 것 같습니다!!
- 항상 통일성 있게 써주셔야합니다!! 제가 반복해서 말씀드리는 부분이 많은데 문제 푸실 때 이런 기본적인 컨벤션 검토나 통일성은 제가 계속계속 얘기해도 본인이 신경쓰셔야 변합니다~!! 🙇‍♂️

Reply:
컨벤션 부분을 전혀 고려안하고 풀었던 것 같습니다... 유의하면서 풀이 진행하겠습니다 감사합니다! ㅠㅠ